tests/global_config_gcnv.py

```
@@ -0,0 +1,37 @@
+
+for _dir in dir():
+    if not config:  # noqa: F821
+        config: dict[str, Any] = {}
+    val = locals()[_dir]
+    if type(val) not in [bool, list, dict, str]:
+        continue
+
+    if _dir in ["encoding", "py_file"]:
+        continue
+
+    config[_dir] = locals()[_dir]  # noqa: F821
```

Ruff (0.14.10)

Unused `noqa` directive (unused: `F821`)

Remove unused `noqa` directive

(RUF100)